### PR TITLE
Document legacy code in activity diagrams

### LINKED ISSUE #72: Create documentation for existing low voltage generation
Introduce RTD support and create a flow chart using PlantUML of the existing algorithm to create low voltage grids

- [x] add RTD support to the repo 
- [x] create flow charts of the existing low voltage grid algorithms

### PR DESCRIPTION
**Pull request by ckittl**
Resolves #72

### CODE REVIEW

**Comment by johanneshiry** [comment]
Does it make sense to add RTD support directly in the context of this PR? Would appreciate to have a consist documentation which (for now) also includes the legacy stuff, as it is then part of the documentation that will be available in the future and can be easily trace back :-)

**Comment by ckittl** [comment]
:thinking: Wanted to introduce it with #75, but can do it here as well.

**Comment by ckittl** [comment]
!test

**Comment by ckittl** [comment]
Better, @johanneshiry ?

File: docs/readthedocs/conf.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'recommonmark',
+    'sphinx.ext.autosectionlabel'
+]
```
Comment (t-ober) [inline docs/readthedocs/conf.py:41]:
```suggestion
extensions = [
    'myst_parser',
    'sphinx.ext.autosectionlabel',
]
```

Comment (t-ober) [reply docs/readthedocs/conf.py:41]: as recommonmark is deprecated (https://github.com/readthedocs/recommonmark)

**Review by t-ober — CHANGES_REQUESTED** [review]

**Review by t-ober — APPROVED** [review]

**Pull request merged**